frontend/src/lib/SessionInfo.ts

```diff
@@ -110,7 +109,6 @@ export class SessionInfo {
       pythonVersion: environmentInfo.pythonVersion,
       installationId: userInfo.installationId,
       installationIdV3: userInfo.installationIdV3,
-      authorEmail: userInfo.email,
```

Comment:
I think there are also a few aspects on the python side that we can remove related to the `email`. For example:

https://github.com/streamlit/streamlit/blob/d9920c82d875afdeae38931984b7a2b496e4a482/proto/streamlit/proto/NewSession.proto#L161

and 

https://github.com/streamlit/streamlit/blob/d9920c82d875afdeae38931984b7a2b496e4a482/lib/streamlit/runtime/app_session.py#L792

Reply:
@LukasMasuch I feel a bit less confident about being sure what all those touch (e.g. that would seem to affect app_session_test)-- if someone else would be willing to remove those in a separate PR, that would be appreciated

Reply:
Yes, that is what I was seeing, and assumed it was related 😂

Reply:
I started a fast follow PR [here](https://github.com/streamlit/streamlit/pull/6516)